pulp_2to3_migration/app/models/content.py

```diff
@@ -74,3 +74,27 @@ def create_pulp3_content(self):
         Return an unsaved Pulp 3 Content
         """
         raise NotImplementedError()
+
+
+class Pulp2LazyCatalog(Model):
+    """
+    Information for downloading Pulp 2 on_demand content.
+
+    Fields:
+        pulp2_importer_id (models.CharField): Importer ID in Pulp 2
+        pulp2_unit_id (models.CharField): Content unit ID in Pulp 2
```

Comment:
Is this stored as a uuid in Pulp 2, or are mongodb IDs their own special thing?  (bson ID or something).

If UUID then we could store it as such.

Reply:
@dralley, I think they are uuids. And on the content model I initially stored it like uuid. 
I ended up using just a char field to avoid conversion from uuid to string and back.
I don't remember where it was, probably if you search in mongo by this field. I can check or try to switch to it.
FWIW, this only used for mongo and uniquness constraints in pre-migrated models, this value is not migrated to Pulp 3.

Reply:
It's unnecessary.  I was just thinking that a native uuid type would probably be a little faster and save some space in the DB/memory.  But not a big deal in either case, especially if this way has cleaner code.